scripts/init-piwik.sh

```diff
@@ -64,48 +78,54 @@ fi
 echo "Done setting up piwik config..."
 cat $CONFIG_FILE
 chown www-data:www-data $CONFIG_FILE
-
-if [ ! -z $PIWIK_SEED_DATABASE ]; then
-  mysql --user=$DB_USER --password=$DB_PASSWORD --host=$DB_HOST \
-        --port=$DB_PORT -D $DB_NAME < /usr/share/nginx/html/config/base-schema.sql
+if [ ! -z $NEW_INSTALL ]; then
+	if [ ! -z $PIWIK_SEED_DATABASE ]; then
+	  mysql --connect-timeout=20 --user=$DB_USER --password=$DB_PASSWORD --host=$DB_HOST \
+		--port=$DB_PORT -D $DB_NAME < /usr/share/nginx/html/config/base-schema.sql
+	  if [ $? -eq 1 ]; then
+	    sleep 20
+	    mysql --connect-timeout=20 --user=$DB_USER --password=$DB_PASSWORD --host=$DB_HOST \
+		--port=$DB_PORT -D $DB_NAME < /usr/share/nginx/html/config/base-schema.sql
+	  fi
   SITE_SQL=$(cat <<EOF
-        INSERT INTO \`piwik_site\`
-        VALUES (1,
-                'Example Piwik Site',
-                'http://www.example.com',
-                '2014-11-01 12:00:00',
-                0,
-                1,
-                '',
-                '',
-                'UTC',
-                'USD',
-                '',
-                '',
-                '',
-                '',
-                'website',
-                0);
+	INSERT INTO \`piwik_site\`
+	VALUES (1,
+		'Example Piwik Site',
+		'http://www.example.com',
+		'2014-11-01 12:00:00',
+		0,
+		1,
+		'',
+		'',
+		'Europe/Paris',
```

Comment:
Hi Brian,

About the exposed ports, yes, it was not an issue, your syntax is fine, I
changed it for some tests and forgot to put it back.

For the timzone, sorry again, I forgot to put back the default values. Same
thing for the charset.

About the Piwik version, your argument is good, let's do it as you said !

Benoît Pourre
Tel: (+33)6 81 36 78 92

2015-05-31 21:59 GMT+02:00 Brian Prodoehl <notifications@github.com>:

> In scripts/init-piwik.sh
> <https://github.com/bprodoehl/docker-piwik/pull/3#discussion_r31393737>:
>
> > -                '',
> > -                '',
> > -                '',
> > -                '',
> > -                'website',
> > -                0);
> > +	INSERT INTO \`piwik_site\`
> > +	VALUES (1,
> > +		'Example Piwik Site',
> > +		'http://www.example.com',
> > +		'2014-11-01 12:00:00',
> > +		0,
> > +		1,
> > +		'',
> > +		'',
> > +		'Europe/Paris',
>
> Default site timezone is still UTC above (SitesManager_DefaultTimezone),
> so I guess it doesn't matter if this example site moves to a different
> timezone.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bprodoehl/docker-piwik/pull/3/files#r31393737>.
>